src/styles/Variables.css

```diff
@@ -3,4 +3,5 @@
     --red-color:red;
     --searchBar-tags:rgb(60, 106, 170);
```

Comment:
I think it'd be better to keep these names more general, so you could call it 'ocean blue'